wp-remember-the-galleries/wp-remember-the-galleries.php

```diff
@@ -248,7 +248,7 @@ static function enqueue_scripts() {
 			)
 		);
 
-		if( $current_screen->id === 'upload' || $current_screen->id === 'post' ||
+		if( $current_screen->id === 'upload' || $pagenow === 'post.php' ||
```

Comment:
and `edit.php` is the page which lists posts for editing I believe. Now that I'm thinking about it though, I think on a new post the `$pagenow` is going to be `post-new.php`. Lemme confirm this before merging.